src/main/java/io/lettuce/core/RedisCommandBuilder.java

```diff
@@ -40,14 +27,176 @@
+import static io.lettuce.core.protocol.CommandKeyword.NODE;
+import static io.lettuce.core.protocol.CommandKeyword.NODES;
+import static io.lettuce.core.protocol.CommandKeyword.NOSAVE;
+import static io.lettuce.core.protocol.CommandKeyword.NOT;
+import static io.lettuce.core.protocol.CommandKeyword.NOVALUES;
+import static io.lettuce.core.protocol.CommandKeyword.NUMPAT;
+import static io.lettuce.core.protocol.CommandKeyword.NUMSUB;
+import static io.lettuce.core.protocol.CommandKeyword.OFF;
+import static io.lettuce.core.protocol.CommandKeyword.ON;
+import static io.lettuce.core.protocol.CommandKeyword.ONE;
+import static io.lettuce.core.protocol.CommandKeyword.OR;
+import static io.lettuce.core.protocol.CommandKeyword.PAUSE;
+import static io.lettuce.core.protocol.CommandKeyword.REFCOUNT;
+import static io.lettuce.core.protocol.CommandKeyword.RELOAD;
+import static io.lettuce.core.protocol.CommandKeyword.REPLACE;
+import static io.lettuce.core.protocol.CommandKeyword.REPLICAS;
+import static io.lettuce.core.protocol.CommandKeyword.REPLICATE;
+import static io.lettuce.core.protocol.CommandKeyword.RESET;
+import static io.lettuce.core.protocol.CommandKeyword.RESETSTAT;
+import static io.lettuce.core.protocol.CommandKeyword.RESTART;
+import static io.lettuce.core.protocol.CommandKeyword.REV;
+import static io.lettuce.core.protocol.CommandKeyword.REWRITE;
+import static io.lettuce.core.protocol.CommandKeyword.SAVECONFIG;
+import static io.lettuce.core.protocol.CommandKeyword.SEGFAULT;
+import static io.lettuce.core.protocol.CommandKeyword.SETINFO;
+import static io.lettuce.core.protocol.CommandKeyword.SETNAME;
+import static io.lettuce.core.protocol.CommandKeyword.SETSLOT;
+import static io.lettuce.core.protocol.CommandKeyword.SETUSER;
+import static io.lettuce.core.protocol.CommandKeyword.SHARDCHANNELS;
+import static io.lettuce.core.protocol.CommandKeyword.SHARDNUMSUB;
+import static io.lettuce.core.protocol.CommandKeyword.SHARDS;
+import static io.lettuce.core.protocol.CommandKeyword.SLAVES;
+import static io.lettuce.core.protocol.CommandKeyword.SLOTS;
+import static io.lettuce.core.protocol.CommandKeyword.SOFT;
+import static io.lettuce.core.protocol.CommandKeyword.STABLE;
+import static io.lettuce.core.protocol.CommandKeyword.STREAM;
+import static io.lettuce.core.protocol.CommandKeyword.TAKEOVER;
+import static io.lettuce.core.protocol.CommandKeyword.TRACKING;
+import static io.lettuce.core.protocol.CommandKeyword.UNBLOCK;
+import static io.lettuce.core.protocol.CommandKeyword.USAGE;
+import static io.lettuce.core.protocol.CommandKeyword.USERS;
+import static io.lettuce.core.protocol.CommandKeyword.WHOAMI;
+import static io.lettuce.core.protocol.CommandKeyword.WITHSCORE;
+import static io.lettuce.core.protocol.CommandKeyword.WITHSCORES;
+import static io.lettuce.core.protocol.CommandKeyword.WITHVALUES;
+import static io.lettuce.core.protocol.CommandKeyword.XOR;
+import static io.lettuce.core.protocol.CommandKeyword.YES;
```

Comment:
are we sure making static imports this way ?

Reply:
I had set my IDE to use `*` when we have more than 10 imports.

Reply:
Hm, I thought we wanted to avoid * imports.
I have no personal opinion on the matter, just consistency.

So if we put a star import after the 10th import I will set this up locally too.

Do we want to enforce this for future PRs?